packages/iris-grid/src/sidebar/ChartBuilder.test.jsx

```diff
@@ -24,19 +24,18 @@ it('renders without crashing', () => {
 });
 
 it('updates the chart type', () => {
-  const { container, unmount } = makeChartBuilderWrapper();
-
-  let chartButtons = container.querySelectorAll('.btn-chart-type');
+  const { unmount } = makeChartBuilderWrapper();
 
-  expect(chartButtons[0].classList.contains('active')).toBe(true);
-  expect(chartButtons[1].classList.contains('active')).toBe(false);
+  const lineButton = screen.getByText('Line');
+  const barButton = screen.getByText('Bar');
```

Comment:
I don't think you have to `import screen` here

It looks like render returns queries with their argument bound to the base element [See here](https://testing-library.com/docs/react-testing-library/api/#queries-1)

So instead you can just `const { unmount, getByText } = makeChartBuilderWrapper()`